ask.py

```diff
@@ -0,0 +1,25 @@
+import os
+import openai
+from dotenv import load_dotenv
+
+load_dotenv()
+
+OPENAI_API_KEY = os.getenv("OPENAI_API_KEY")
```

Comment:
you should add the api key to openai configuration. look at the example here: https://github.com/openai/openai-python

Reply:
I have already added it, but still not working:
![image](https://github.com/Amenokal-Labs/legal-ai-assistant/assets/80546947/7ac8d4ac-f2fa-4619-aef1-bd202c2a57c7)